packages/flutter/lib/src/material/typography.dart

```diff
@@ -373,3 +335,150 @@ class Typography {
   /// A material design text theme with light glyphs.
   final TextTheme white;
 }
+
+/// Provides default font family and text colors compliant with the Material
+/// Design specification.
+///
+/// The geometric properties provided here use values for English-like category
+/// of languages meant to be used as defaults and fallbacks. Apps distributed
+/// internationally are expected to use [MaterialLocalizations], which, together
+/// with [Theme.of], localizes text geometry for other language categories.
```

Comment:
are these fontSizes/fontWeights/baselines ever actually used? Can we get away with not specifying them?

Reply:
Depends on what you mean by "actually used". Today we support using `Typography` without the `Theme`. Example:

```dart
  testWidgets('geometry defaults are used', (WidgetTester tester) {
    final Typography typography = new Typography(platform: TargetPlatform.android);
    tester.pumpWidget(new Directionality(
      textDirection: TextDirection.ltr,
      child: new Text('Hello', style: typography.white.body1),
    ));
  });
```

I'm not sure if _our_ sample apps use it in this way. Probably rarely or not at all.

Reply:
Removed fontSizes/fontWeights/baselines in https://github.com/flutter/flutter/pull/11829/commits/98612e0112640c0f6148816556f32175d72f4bab. Wasn't a big deal for our tests.